tests/acceptance/pageObjects/FilesPageElement/publicLinksDialog.js

```diff
@@ -488,7 +488,10 @@ module.exports = {
         .setValue('@dialogInput', password)
         .useCss()
 
-      await this.click('@dialogConfirmBtnEnabled')
+      await this.waitForElementVisible('@dialogConfirmBtnEnabled')
+        .initAjaxCounters()
+        .click('@dialogConfirmBtnEnabled')
+        .waitForOutstandingAjaxCalls()
```

Comment:
@pascalwengerter I have made these changes. IMO this might fix the `webUISharingPublicDifferentRoles/shareByPublicLinkDifferentRoles.feature:27{6,8}` failure.